password-management.md

```diff
@@ -0,0 +1,422 @@
+---
+title: TiDB Password Management
+summary: Learn the mechanism of user password management in TiDB.
+---
+
+# TiDB Password Management
+
+To protect the security of user passwords, TiDB supports the following password management policies starting from v6.5.0:
+
+- Password complexity policy: require users to set strong passwords to prevent empty and weak passwords.
+- Password expiration policy: require users to change their passwords periodically.
+- Password reuse policy: prevent users from reusing old passwords.
+- Failed-login tracking and temporary account locking: temporarily lock the user account and prevent the same user from trying to login after multiple login failures caused by wrong passwords.
+
+## TiDB authentication credential storage
+
+To ensure the authenticity of user identify, TiDB uses password as a credential to authenticate users when they login in to the TiDB server.
+
+The *password* described in this document refers to the internal credentials generated, stored, and verified by TiDB. TiDB stores user passwords in the `mysql.user` system table.
+
+The following authentication plugin is related to TiDB password management:
+
+- `mysql_native_password`
+- `caching_sha2_password`
+- `tidb_sm3_password`
+
+For more information about TiDB authentication plugins, see [Authentication plugin status](/security-compatibility-with-mysql.md#authentication-plugin-status).
+
+## Password complexity policy
+
+Password complexity check is disabled by default in TiDB. By configuring system variables related to password complexity, you can enable the password complexity check and make sure the user passwords comply with the password complexity policy.
+
+The password complexity policy has the following features:
+
+- For SQL statements that sets user passwords in plaintext (including `CREATE USER`, `ALTER USER`, and `SET PASSWORD`), TiDB checks the passwords against the password complexity policy. If the password does not meet the requirements, the password is rejected.
+- You can use the SQL function [`VALIDATE_PASSWORD_STRENGTH()`](https://dev.mysql.com/doc/refman/5.7/en/encryption-functions.html#function_validate-password-strength) to validate the password strength.
+
+> **Note:**
+>
+> - For the `CREATE USER` statement, even if the account is locked upon creation, you must set an acceptable password. Otherwise, when the account is unlocked, this account can log in to TiDB using a password that does not comply with the password complexity policy.
```

Comment:
```suggestion
> - For the `CREATE USER` statement, even if you can lock the account upon creation, you must set an acceptable password. Otherwise, when the account is unlocked, this account can log in to TiDB using a password that does not comply with the password complexity policy.
```